MAINTAINERS.md

```diff
@@ -0,0 +1,197 @@
+# Maintainers
+
+This document serves to clarify the responsibilities of a maintainer as well as
+serve as a wiki for maintainers to link to / include resources.
+
+
+## Current Maintainers
+
+The current maintainers are:
+
+| name | focus areas |
+------------------
+|   @achow101  | wallet, PSBTs |
+|   @fanquake | general, tooling, build system |
+|   @hebasto | UI/UX |
+|   @laanwj | general |
+|   @marcofalke | general |
```

Comment:
IIRC, Marco's focus was tests/QA?

Reply:
ah i see -- I wasn't quite sure about this section so I mostly guessed.

@MarcoFalke is that accurate? would it be better to be general + tests/QA, or just tests/QA?